### PR TITLE
Storybook fix

### LINKED ISSUE #112: Storybook not running in dev
There's an issue with the decorators in Storybook - you can build and it runs fine, but running `yarn storybook` will give you this:

image

The quick work around for that is to change the import in `client/packages/common/src/intl/index.ts` from

```
export { LocaleKey } from './locales';
```

to 

```
import { LocaleKey } from './locales';
export { LocaleKey };
```

which then gives the next issue. Basically, there is a problem with imports in `<StoryProvider>` when used within `preview.js`

### PR DESCRIPTION
**Pull request by mark-prins**
Fixes #112 

It's working!

### CODE REVIEW

File: client/.storybook/main.js [hunk 1]
```diff
@@ -5,12 +5,14 @@ const toPath = filePath => path.join(process.cwd(), filePath);
...
   typescript: { reactDocgen: 'react-docgen' },
-  reactOptions: {
-    fastRefresh: true,
-  },
   framework: '@storybook/react',
   core: {
-    builder: '@storybook/builder-webpack5',
+    builder: {
+      name: 'webpack5',
+      options: {
```
Comment (mark-prins) [inline client/.storybook/main.js:12]: not part of the fix, and an experimental feature of webpack. but neat - speeds up compilation time in develop mode, at the expense of slightly longer load times, most noticeable if you're switching between different stories

File: client/.storybook/main.js [hunk 2]
```diff
@@ -20,23 +22,6 @@ module.exports = {
     '@storybook/addon-links',
     '@storybook/addon-essentials',
     '@storybook/addon-interactions',
```
Comment (mark-prins) [inline client/.storybook/main.js:24]: the storybook-addon is part of the issue, and is not required

File: client/.storybook/main.js [hunk 3]
```diff
@@ -5,12 +5,14 @@ const toPath = filePath => path.join(process.cwd(), filePath);
 module.exports = {
   staticDirs: ['../packages/host/public'],
   typescript: { reactDocgen: 'react-docgen' },
```
Comment (mark-prins) [inline client/.storybook/main.js:7]: the fast refresh is part of the problem

File: client/package.json [hunk 4]
```diff
@@ -41,18 +41,18 @@
...
     "@graphql-codegen/typescript-operations": "^2.2.1",
-    "@storybook/addon-actions": "^6.5.8",
-    "@storybook/addon-essentials": "^6.5.8",
-    "@storybook/addon-interactions": "^6.5.8",
-    "@storybook/addon-links": "^6.5.8",
-    "@storybook/addons": "^6.5.8",
-    "@storybook/builder-webpack5": "^6.5.8",
-    "@storybook/manager-webpack5": "^6.5.8",
-    "@storybook/react": "^6.5.8",
+    "@storybook/addon-actions": "^6.5.9",
```
Comment (mark-prins) [inline client/package.json:44]: not strictly necessary, and not a solution in itself. but why not?

**Review by CarlosNZ — APPROVED** [review]
Awesome, good to have it back at last! :)

**Pull request merged**